plugin-core/src/main/scala/com/alpine/plugin/core/OperatorParameters.scala

```diff
@@ -89,4 +89,5 @@ trait OperatorParameters {
    */
   def getParameterIds: Iterator[String]
 
+  def getAdvancedSparkParameters: scala.collection.mutable.Map[String, String]
```

Comment:
I do not recall there being any code that depends on this being mutable. I can change it.